packages/core/src/components/date-picker/date-picker.stories.ts

```diff
@@ -8,6 +10,14 @@ import {
 } from './date-picker.template';
 import readme from './readme.md';
 
+function decorator(story: ArgsStoryFn<TemplateResult>): TemplateResult {
```

Comment:
In de React implementatie heb je de decorator in z'n eigen file gestopt, ik vind dat wel fraai. Waarom heb je dat hier niet gedaan?

Reply:
OK, goed punt. Dan pakken we dit bij #1438 mee.